pkg/api/universal/conversation_stream.go

```diff
@@ -0,0 +1,312 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package universal
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	piiscrubber "github.com/aavaz-ai/pii-scrubber"
+
+	"github.com/dapr/components-contrib/conversation"
+	diag "github.com/dapr/dapr/pkg/diagnostics"
+	"github.com/dapr/dapr/pkg/messages"
+	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
+	"github.com/dapr/kit/logger"
+	kmeta "github.com/dapr/kit/metadata"
+)
+
+// defaultPIIScrubberWindowSize is the default window size (bytes) for PII scrubbing
+// this allows for a look-ahead window of stream data to be scrubbed
+// TODO: make this configurable
+const defaultPIIScrubberWindowSize = 50
+
+// ConverseStreamAlpha1 implements the streaming conversation API
+func (a *Universal) ConverseStreamAlpha1(req *runtimev1pb.ConversationRequest, stream runtimev1pb.Dapr_ConverseStreamAlpha1Server) (retErr error) {
```

Comment:
Please do not use return values for least surprising. It makes it more difficult to read the function contents.

Name should at least be `error`.

```suggestion
func (a *Universal) ConverseStreamAlpha1(req *runtimev1pb.ConversationRequest, stream runtimev1pb.Dapr_ConverseStreamAlpha1Server) error {
```